docs/support_table.md

```diff
@@ -0,0 +1,1173 @@
+|\smile|$\smile$||
+|\smiley|<span style="color:firebrick;">Not supported</span>||
+|\sout|$\sout{abc}$|`\sout{abc}`|
+|\Space|<span style="color:firebrick;">Not supported</span>|see `\space`
+|\space|$a\space{b}$|`a\space{b}`|
+|\spades|<span style="color:firebrick;">Not supported</span>||
+|\spadesuit|$\spadesuit$||
+|\sphericalangle|$\sphericalangle$||
+|{split}|<span style="color:firebrick;">Not supported</span>||
+|\sqcap|$\sqcap$||
+|\sqcup|$\sqcup$||
+|\square|$\square$||
+|\sqrt|$\sqrt[3]{x}$|`\sqrt[3]{x}`|
+|\sqsubset|$\sqsubset$||
+|\sqsubseteq|$\sqsubseteq$||
+|\sqsupset|$\sqsupset$||
+|\sqsupseteq|$\sqsupseteq$||
+|\ss|$\text{\ss}$|`\text{\ss}`|
+|\stackrel|$\stackrel{!}{=}$|`\stackrel{!}{=}`|
+|\star|$\star$||
+|\Stigma|<span style="color:firebrick;">Not supported</span>||
+|\stigma|<span style="color:firebrick;">Not supported</span>||
+|\strut|<span style="color:firebrick;">Not supported</span>||
+|\style|<span style="color:firebrick;">Not supported</span>|Non standard|
+|\SUB|<span style="color:firebrick;">Not supported</span>||
+|{subarray}|<span style="color:firebrick;">Not supported</span>||
+|\sube|<span style="color:firebrick;">Not supported</span>||
+|\Subset|$\Subset$||
+|\subset|$\subset$||
+|\subseteq|$\subseteq$||
+|\subseteqq|$\subseteqq$||
+|\subsetneq|$\subsetneq$||
+|\subsetneqq|$\subsetneqq$||
+|\substack|<span style="color:firebrick;">Not supported</span>||
+|\succ|$\succ$||
+|\succapprox|$\succapprox$||
+|\succcurlyeq|$\succcurlyeq$||
+|\succeq|$\succeq$||
+|\succnapprox|$\succnapprox$||
+|\succneqq|$\succneqq$||
+|\succnsim|$\succnsim$||
+|\succsim|$\succsim$||
+|\sum|$\sum$||
+|\sup|$\sup$||
+|\supe|<span style="color:firebrick;">Not supported</span>||
+|\Supset|$\Supset$||
+|\supset|$\supset$||
+|\supseteq|$\supseteq$||
+|\supseteqq|$\supseteqq$||
+|\supsetneq|$\supsetneq$||
+|\supsetneqq|$\supsetneqq$||
+|\surd|$\surd$||
+|\swarrow|$\swarrow$||
+
+## T
+
+|Symbol/Function |  Rendered   | Source or Comment|
+|:---------------|:------------|:-----------------|
+|\tag|$$\tag{3.1c} a^2+b^2=c^2$$|`\tag{3.1c} a^2+b^2=c^2`|
+|\tag*|$$\tag*{3.1c} a^2+b^2=c^2$$|`\tag*{3.1c} a^2+b^2=c^2`|
+|\tan|$\tan$||
+|\tanh|$\tanh$||
+|\Tau|<span style="color:firebrick;">Not supported</span>||
+|\tau|$\tau$||
+|\tbinom|$\tbinom n k$|`\tbinom n k`|
+|\TeX|$\TeX$||
+|\text|$\text{ yes }\&\text{ no }$|`\text{ yes }\&\text{ no }`|
+|\textasciitilde|$\text{\textasciitilde}$|`\text{\textasciitilde}`|
+|\textbar|$\text{\textbar}$|`\text{\textbar}`|
+|\textbardbl|$\text{\textbardbl}$|`\text{\textbardbl}`|
+|\textbf|$\textbf{AaBb123}$|`\textbf{AaBb123}`|
+|\textbraceleft|$\text{\textbraceleft}$|`\text{\textbraceleft}`|
+|\textbraceright|$\text{\textbraceright}$|`\text{\textbraceright}`|
+|\textcircled|$\text{\textcircled a}$|`\text{\textcircled a}`|
+|\textcolor|$\textcolor{blue}{F=ma}$|`\textcolor{blue}{F=ma}`|
+|\textdagger|$\text{\textdagger}$|`\text{\textdagger}`|
+|\textdaggerdbl|$\text{\textdaggerdbl}$|`\text{\textdaggerdbl}`|
```

Comment:
It seems there is a bug with `\textdaggerdbl`.

Reply:
> It seems there is a bug with \textdaggerdbl.

Yeah, I noticed that, too. I'll open an issue.

It seems that this Support Table will also serve as a test bed. I've found other problems while working on it.